core/tabs/common-script.sh

```diff
@@ -125,11 +125,11 @@ checkDistro() {
 }
 
 checkEnv() {
-    checkCommandRequirements 'curl groups sudo'
+    checkEscalationTool
+    checkCommandRequirements "curl groups $ESCALATION_TOOL"
```

Comment:
```suggestion
    checkCommandRequirements "curl groups"
```
just remove sudo and escalation_tool from here, if you add escalation tool to the command requirements than it will require the user to have both sudo and doas, which we do not want that